### ISSUE TITLE
result contains layouts, pages

### ISSUE THREAD

**Issue by makotot** [makotot/assemble-unused-partials#6]
```
❯ Unused partials
 ⚠ src/layouts/default.hbs
 ⚠ src/pages/fixture-1.hbs
 ⚠ src/pages/index.hbs
 ⚠ src/partials/unused.hbs
 ⚠ src/sub-partials/sub-unused.hbs
```

**Comment by makotot** [comment]
Should be fixed in https://github.com/makotot/get-assemble-partials/issues/1

**Comment by makotot** [comment]
Is not a bug. Fixed in #7 .
